tests/framework/results/test_manager.py

```diff
@@ -301,6 +301,21 @@ def test_stratified_metrics_initialized_as_zeros_dataframes():
     )
 
 
+def test_no_stratified_metrics_initialization():
+    """Test that if no stratifications are registered then we initialize a
+    single-row DataFrame with 'value' of zero and index labeled 'all'
+    """
+    components = [QuidditchWinsObserver(), HousePointsObserver(), Hogwarts()]
```

Comment:
Shouldn't we get an error, since we have observers that are asking for stratifications that haven't been registered?

Reply:
Hmm, I don't think that's a current feature. There's no guarantee at the point of observer registration what stratifications will eventually exist (I think?). Is this something we should be supporting? It could happen in post setup or something.

Reply:
Fair enough.